CLAUDE.md

```diff
@@ -94,6 +94,16 @@ Custom tool definitions in `tools/` package with:
 - `ToolSchemaUtils` for JSON schema generation
 - Function-specific implementations (ReadFileTool, EditFileTool, ListFilesTool)
 
+## Code Style Guidelines
```

Comment:
It's better to move this section between 'Dependencies' and 'Git Commit Guidelines'